celestia-application.md

```diff
@@ -0,0 +1,220 @@
+  status              Query remote node for status
+  tendermint          Tendermint subcommands
+  tx                  Transactions subcommands
+  unsafe-reset-all    Resets the blockchain database, removes address book files, and resets data/priv_validator_state.json to the genesis state
+  validate-genesis    validates the genesis file at the default location or at the location passed as an arg
+  version             Print the application binary version information
+
+Flags:
+  -h, --help                help for celestia-appd
+      --home string         directory for config and data (default "/home/pops/.celestia-app")
+      --log_format string   The logging format (json|plain) (default "plain")
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "celestia-appd [command] --help" for more information about a command.
+```
+## Setting up Network
+First clone the networks repository:
+```sh
+cd $HOME
+rm -rf networks
+git clone https://github.com/celestiaorg/networks.git
+```
+To initialize the network pick a "node-name" that describes your node. The --chain-id parameter we are using here is "devnet-2". Keep in mind that this might change if a new testnet is deployed.
+```sh
+celestia-appd init "node-name" --chain-id devnet-2
+```
+Copy the `genesis.json` file. For devnet-2 we are using:
+```sh
+cp $HOME/networks/devnet-2/genesis.json $HOME/.celestia-app/config
+```
+Set seeds and peers:
+```sh
+SEEDS="74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656"
+PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
+```
+Reset network:
+```sh
+celestia-appd unsafe-reset-all
+```
+## Running Ceslestia-App using Systemd
+Create Celestia-App systemd file:
+```sh
+sudo tee <<EOF >/dev/null /etc/systemd/system/celestia-appd.service
+[Unit]
+Description=celestia-appd Cosmos daemon
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=$HOME/go/bin/celestia-appd start
+Restart=on-failure
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target
+EOF
+```
+If the file was created succesfully you will be able to see its content:
+```sh
+cat /etc/systemd/system/celestia-appd.service
+```
+Enable and start celestia-appd daemon:
+```sh
+sudo systemctl enable celestia-appd
+sudo systemctl start celestia-appd
+```
+Check if daemon has been started correctly:
+```sh
+sudo systemctl status celestia-appd
+```
+Check daemon logs in real time:
+```sh
+sudo journalctl -u celestia-appd.service -f
+```
+To check if your node is in sync before going forward:
+```sh
+curl -s localhost:26657/status | jq .result | jq .sync_info
+```
+Make sure that you have `"catching_up": false`, otherwise leave it running until it is in sync.
+
```

Comment:
we need 2 more sections here: 
1. Faucet 
2. How to delegate stake to the active validator set 

Afterwards, let's point the user to the celestia-node.md part with the message like "Now, you can go and install/run a Celestia Full node"

Reply:
So, I added a How to Delegate Stake to a Validator section. I am not quite sure about the faucet part, since funding a wallet is already included inside the Validator Creation section. Maybe you mean to add these steps earlier so those who don't want to create a validator can interact with the node?

Reply:
>  Maybe you mean to add these steps earlier so those who don't want to create a validator can interact with the node?

Great question! Yes, it is better to add the steps earlier in the non-validating node setup chapter, rather then in the validator one

So you will have the struct of the file looking something like: 

1. Running a Non-Validating Celestia Application Node
2. Installing Dependencies
3. Installing GO
4. Downloading and Compiling Celestia-App
5. Setting up Network
6. Running Ceslestia-App using Systemd
7. Creating the wallet 
8. Funding the wallet
8. Delegating the stake
9. Go to celestia-node.md part 

Then for the validator part we only need tthe chapter  "Creating the Validator On-Chain"

Reply:
So, I changed the order of the menu/content accordingly. Also linked the celestia-node.md#running-a-celestia-node at the very end in case the user wants to create a Celestia node.